### ISSUE TITLE
If env AWS_PROFILE or AWS_DEFAULT_PROFILE are set, a nonsensical provider_id is used.

### ISSUE THREAD

**Issue by eviln1** [venth/aws-adfs#60]
Hi,
```
[user@host ~]$ aws-adfs --version
0.3.14
[user@host ~]$ aws --version
aws-cli/1.11.154 Python/3.6.2 Linux/4.13.12-200.fc26.x86_64 botocore/1.7.12
```

I have noticed that `aws-adfs` breaks when env var `AWS_PROFILE` (or `AWS_DEFAULT_PROFILE`) are set. `--debug` shows that the value of `AWS_PROFILE` replaces `provider_id`, which is used to construct the `_IDP_ENTRY_URL` : 
https://github.com/venth/aws-adfs/blob/8ded19f79abac8957129c45cafb1465c740e563e/aws_adfs/prepare.py#L91-L92

I am not sure that `boto.session`'s value `profile` and `provider_id` are interchangeable...

**Comment by venth** [comment]
From the [documentation](http://boto.cloudhackers.com/en/latest/ref/sts.html):
```
provider_id (string) – Specify this value only for OAuth access tokens. Do not specify this value for OpenID Connect ID tokens, such as accounts.google.com. This is the fully-qualified host component of the domain name of the identity provider. Do not include URL schemes and port numbers. Currently, www.amazon.com and graph.facebook.com are supported.
```

It seems that I wrongly provide there aws profile.

**Comment by venth** [comment]
Fix released in version: 0.7.0
